### ISSUE TITLE
Automatic error report

### ISSUE THREAD

**Issue by esunny** [spyder-ide/spyder#7891]
### What steps will reproduce the problem?

1.) Installed Anaconda
2.) Launched Anaconda
3.) Tried to launch Spyder
4.) Error window poped out.

### Traceback
```python-traceback
  File "/Users/eSunny/anaconda2/lib/python2.7/site-packages/qtconsole/base_frontend_mixin.py", line 163, in _dispatch
    handler(msg)
  File "/Users/eSunny/anaconda2/lib/python2.7/site-packages/spyder/widgets/ipythonconsole/namespacebrowser.py", line 218, in _handle_execute_reply
    self.refresh_namespacebrowser()
  File "/Users/eSunny/anaconda2/lib/python2.7/site-packages/spyder/widgets/ipythonconsole/namespacebrowser.py", line 67, in refresh_namespacebrowser
    'get_ipython().kernel.get_namespace_view()')
  File "/Users/eSunny/anaconda2/lib/python2.7/site-packages/spyder/widgets/ipythonconsole/shell.py", line 341, in silent_exec_method
    local_uuid = to_text_string(uuid.uuid1())
  File "/Users/eSunny/anaconda2/lib/python2.7/uuid.py", line 588, in uuid1
    clock_seq_hi_variant, clock_seq_low, node), version=1)
  File "/Users/eSunny/anaconda2/lib/python2.7/uuid.py", line 164, in __init__
    raise ValueError('field 6 out of range (need a 48-bit value)')
ValueError: field 6 out of range (need a 48-bit value)

**Comment by ccordoba12** [comment]
This is an error in Python itself, fixed in Python 3.6.5 and not backported to Python 2.

So I'm afraid you'll have to update to use Python 3.

**Comment by esunny** [comment]
Thank you for the reply :)
But what exactly the error say? Right now I wish to gets hands on Spyder, don't want to change the python to 3.
For time being I've disabled error prompt in Spyder.
Cheers!!
